.github/workflows/ci-build.yml

```diff
@@ -25,6 +26,10 @@ jobs:
           "--features=prio2,multithreaded",
           "--features=prio2,test-util,multithreaded",
         ]
+        packages: [
```

Comment:
Only one of the commands below makes use of this package specifier. As a result, we will have two jobs for each feature, largely performing duplicative work. I think we should take this out of the matrix, and run `cargo build -p prio ...` and `cargo build -p prio-binaries ...` as separate steps.

Reply:
We have to do this anyway because `prio-binaries` has no features.